bionty/base/__init__.py

```diff
@@ -1,10 +1,36 @@
-"""Bionty base.
+"""Bionty base is a standalone version of Bionty without lamindb support.
 
-Import the package::
+Overview
+========
 
-   import bionty.base as bt
+Bionty base is designed for scripts that need to access public ontologies without more advanced lamindb support.
```

Comment:
"Bionty base" is not a name. I'd consistently refer to it as `bionty.base`. If we really want something else, it could be "Bionty Base" or "BiontyBase". But for simplicity I'd just stick with `bionty.base` in verbatim notation.

Reply:
How about: "`bionty.base` is the read-only interface for public ontology that underlies `bionty` and doesn't require a `lamindb` instance."